cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -701,6 +715,7 @@ func main() {
 	listen := flag.String("listen", ":6072", "listen on this address.")
 	hostname := flag.String("hostname", hostnameBestEffort(), "the name we advertise to Sourcegraph when asking for the list of repositories to index. Can also be set via the NODE_NAME environment variable.")
 	cpuFraction := flag.Float64("cpu_fraction", 1.0, "use this fraction of the cores for indexing.")
+	indexingMetricsReposAllowlist := flag.String("INDEXING_METRICS_REPOS_ALLOWLIST", os.Getenv("INDEXING_METRICS_REPOS_ALLOWLIST"), "comma separated list of repositories that we capture separate indexing metrics for")
```

Comment:
is it useful to support this as a flag as well? In practice we only ever configure with envvars since it is a pain to change the args in the docker images we build. But I suppose this is consistent with everything else :)

Reply:
sure - I ripped out the flag in my latest commit